physical/zookeeper/zookeeper.go

```diff
@@ -18,8 +18,8 @@ import (
 	"github.com/hashicorp/vault/sdk/physical"
 
```

Comment:
Can you remove this blank line? I'm curious why go-fumpt didn't do this automatically.